exercise4/removeDuplicates.js

```diff
@@ -0,0 +1,3 @@
+function removeDuplicates(array) {
+  return [...new Set(array)];
```

Comment:
Using a Set is an efficient way to remove duplicates, as Sets automatically enforce uniqueness. Looks good tho.